### ISSUE TITLE
In "grouped" mode (RAW + jpg), selecting one image makes 2 images be selected

### ISSUE THREAD

**Issue by Pointy** [darktable-org/darktable#1937]
I *think* this is a new behavior with 2.6, but I'm not 100% sure.  When I've got a folder full of pairs of RAW and JPEG images and the "G" (grouped) button is clicked, it used to be the case that selecting just one of them would mean that an export would export 1 image. However with 2.6 I notice that selecting one image causes the Lighttable view to say at the top that I've got *two* images selected, and an "export" operation claims that it's doing two exports. (Two separate export messages are logged to `stdout` at that point too.)

Shouldn't it only export 1 image (the processed RAW image) when I've only got one selected?

I'm on Ubuntu 18.04 for what that's worth; I just upgraded to 2.6 this afternoon.

**Comment by elman22** [comment]
Yes, this seems to be bug for me as well. Especially when export exports both images as JPG and I have no idea which one was developed from RAW until I actually open the image and take a look at it.

**Comment by Pointy** [comment]
I've learned since I opened this that the issue is indeed related to the desire to have ratings apply uniformly to group members. I have also gained understanding of the Lighttable images filter mechanism. In my own workflow I generally don't need camera JPG files in Darktable anyway, so I could solve my problems by simply not importing them in the first place. However now that I realize I can filter the Lighttable view by film roll *and* by group rank ("leaders"), things work out perfectly well for me.

I won't retract my "bug" because I think it might be useful as evidence of a need for more clarity in some part of the GUI documentation, though of course I won't have my feelings hurt if it's closed as a "works as intended".
